packages/admin/admin/src/form/FinalFormFileSelect.tsx

```diff
@@ -22,115 +23,204 @@ export type FinalFormFileSelectClassKey =
-            gap: "10px",
-            width: "100%",
-        },
-        droppableArea: {
-            position: "relative",
-            display: "flex",
-            height: "80px",
-            justifyContent: "center",
-            alignItems: "center",
-            alignSelf: "stretch",
-            borderRadius: "4px",
-            border: `1px dashed ${palette.grey[200]}`,
-            cursor: "pointer",
-            "&$droppableAreaIsDisabled": {
-                cursor: "default",
-            },
-            "&$droppableAreaHasError": {
-                border: `1px dashed ${palette.error.main}`,
-            },
-        },
-        droppableAreaCaption: {
-            padding: "30px",
-            color: palette.grey[400],
-        },
-        droppableAreaError: {
-            position: "absolute",
-            top: 0,
-            right: 0,
-            padding: "10px",
-            color: palette.error.main,
-        },
-        fileList: {
-            display: "flex",
-            flexDirection: "column",
-            alignItems: "flex-start",
-            gap: "2px",
-            width: "100%",
-        },
-        fileListItem: {
-            display: "flex",
-            padding: "8px 7px 8px 15px",
-            alignItems: "center",
-            gap: "10px",
-            borderRadius: "4px",
-            background: palette.grey[50],
-            justifyContent: "space-between",
-            width: "100%",
-            boxSizing: "border-box",
-        },
-        fileListItemInfos: {
-            display: "flex",
-            justifyContent: "end",
-            gap: "10px",
-        },
-        rejectedFileListItem: {
-            display: "flex",
-            padding: "14px 15px",
-            alignItems: "center",
-            gap: "10px",
-            borderRadius: "4px",
-            background: palette.grey[50],
-            justifyContent: "space-between",
-            border: `1px dashed ${palette.error.main}`,
-            color: palette.error.main,
-            width: "100%",
-            boxSizing: "border-box",
-        },
-        errorMessage: {
-            display: "flex",
-            alignItems: "center",
-            color: palette.error.main,
-            gap: "5px",
-        },
-        droppableAreaIsDisabled: {},
-        droppableAreaHasError: {},
-        fileListText: {
-            textOverflow: "ellipsis",
-            overflow: "hidden",
-            whiteSpace: "nowrap",
-        },
-        selectButton: {
-            backgroundColor: palette.grey[800],
-            color: palette.common.white,
-            "&:hover": {
-                backgroundColor: palette.grey[800],
-                color: palette.common.white,
-            },
-        },
-    });
-};
+type OwnerState = { droppableAreaIsDisabled: boolean; droppableAreaHasError: boolean };
+
+const Root = createComponentSlot("div")<FinalFormFileSelectClassKey>({
+    componentName: "CometAdminFinalFormFileSelect",
```

Comment:
`CometAdmin` is automatically added when using `createComponentSlot()`

```suggestion
    componentName: "FinalFormFileSelect",
```